casbin-ruby-sql-adapter.gemspec

```diff
@@ -13,9 +13,15 @@ Gem::Specification.new do |s|
   s.licenses    = ['Apache License 2.0']
   s.description = 'SQl adapter for Ruby'
   s.summary     = 'SQl adapter in Ruby'
+  s.files = %w[README.md] + Dir.glob(File.join('lib', '**', '*.rb'))
+  s.test_files = Dir.glob(File.join('spec', '**', '*.rb'))
   s.required_ruby_version = '>= 2.5.0'
 
+  s.add_dependency 'casbin-ruby'
```

Comment:
I think, we should use specific version and use conditions for it like ` 'casbin-ruby', '>= 1.0.7'`